a code review ui that may look familiar to some

jdhollen opened the pull request:
I don't use graphite, so I'm just blasting this out as a change that includes the server changes in #5653

I'll merge that first and bring this one up to it

It looks like this:
<img width="1261" alt="Screenshot 2024-01-04 at 7 24 10 PM" src="https://github.com/buildbuddy-io/buildbuddy/assets/10570856/52464c22-9cb2-4f0b-bccc-9343a8138851">

bduffany commented on enterprise/app/review/view_pull_request.tsx:
```diff
@@ -0,0 +1,390 @@
+      .then((r) => {
+        console.log(r);
+        this.setState({ response: r });
+      });
```
`.catch(e => error_service.handleError(e))`

jdhollen replied:
Oops, thanks

bduffany reviewed (commented):
Looks awesome! :star_struck: 

A couple of UI nits (doesn't have to be in this PR)

- Maybe render `Change 123` as `Pull request #123` in the title bar
- Maybe render the pending status checks with a blue ("in progress") BG to differentiate them from the completed status checks that are grey? Or make the completed ones green so that it matches our UI status color when clicking through to the workflow invocation
- Render the repo name somewhere?

siggisim commented:
> Maybe render Change 123 as Pull request #123 in the title bar
I kind of like using the "Change" terminology. I've always hated "Pull request". If you're a beginner to programming - you know what a change is, but you have no idea what a pull request is.

bduffany commented:
I agree that "pull request" sucks and prefer Gitlab's "Merge request" terminology.
 "Change" is especially confusing in git terms, because it can refer to either a commit, or a request to merge a commit. But given the context of getting here from the "reviews" page it's probably fine.

I do think the "#" would be good to add though because that's usually how it's presented in other UIs

jdhollen commented:
> Looks awesome! 🤩
> 
> A couple of UI nits (doesn't have to be in this PR)
> 
> * Maybe render `Change 123` as `Pull request #123` in the title bar
> * Maybe render the pending status checks with a blue ("in progress") BG to differentiate them from the completed status checks that are grey? and make the completed ones green so that it matches our UI status color when clicking through to the workflow invocation. Currently we use grey for "disconnected" so it feels a bit confusing (I understand the "familiar" UI doesn't use these conventions :P)
> * Render the repo name somewhere?

There will be many more changes to all of these, so I'm going to punt on the bigger tweaks, but i've updated the text in a few places per your comments (and added a doc title, which i forgot to do)

jdhollen commented:
> I agree that "pull request" sucks and prefer Gitlab's "Merge request" terminology. "Change" is ambiguous in git terms, because it can refer to either a commit, or a request to merge a commit. But given the context of getting here from the "reviews" page it's probably fine.
> 
> I do think the "#" would be good to add though because that's usually how it's presented in other UIs

gonna let this relgious battle play out a little longer, we've currently got both on the homepage:
<img width="317" alt="Screenshot 2024-01-05 at 10 22 53 AM" src="https://github.com/buildbuddy-io/buildbuddy/assets/10570856/f20a8a99-3d86-42f8-98d6-20998a78e216">

bduffany commented:
>  we've currently got both on the homepage:

The "PR" bit was my doing :sweat_smile: I shortened it because "Change" was taking up too much space, but maybe "#" or "ID" could also work there?
